### PR TITLE
DOCKER: python script at the end of simulation in BEAM docker image is not working

### PR DESCRIPTION
**Pull request by nikolayilyin**
Currently, python scripts at the end of BEAM simulation are not working because they can not find python modules.
So, I commented they out to reduce image size and image build time.
Investigation of python modules required.

Probably the problem is in using the wrong version of python because for `python` there are no installed modules inside the container but for `python3` all modules are installed correctly.

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3119)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -613,9 +610,15 @@ import com.bmuschko.gradle.docker.tasks.image.DockerTagImage
 task createDockerfile(type: Dockerfile, dependsOn: dockerSyncBuildContext) {
     from 'beammodel/miniconda3'
     label(['maintainer': 'LBNL Beam Team'])
-    instruction 'RUN apt-get update && apt-get -y install python3 python3-pip python3-rtree libproj-dev proj-data proj-bin libgeos-dev npm'
-    instruction 'RUN pip3 install --upgrade setuptools'
-    instruction 'RUN pip3 install numpy pandas cython sklearn matplotlib mapclassify descartes geopandas contextily plotly collections-extended psutil requests'
+
+    // there is a problem with python in the image:
```
Comment (JustinPihony) [inline build.gradle:614]: What was the error? This was originally reviewed and working, so I'd be curious what error was being seen?

Comment (nikolayilyin) [reply build.gradle:614]: the error looks like that:

`14:24:26.231 INFO  beam.analysis.AnalysisProcessor$ - [1] Running python script: src/main/python/events_analysis/analyze_events.py with args WrappedArray(/app/output/sf-light/sf-light-1k-xml__2021-03-03_14-20-35_qzz/ITERS/it.0/0.events.csv.gz)
14:24:26.375 ERROR beam.analysis.AnalysisProcessor$ - [1] Process Handler Stderr for Python Script: src/main/python/events_analysis/analyze_events.py: Traceback (most recent call last):
Process Handler Stdout for Python Script: src/main/python/events_analysis/analyze_events.py: Traceback (most recent call last):
14:24:26.375 ERROR beam.analysis.AnalysisProcessor$ - [2] Process Handler Stderr for Python Script: src/main/python/events_analysis/analyze_events.py:   File "src/main/python/events_analysis/analyze_events.py", line 3, in <module>
Process Handler Stdout for Python Script: src/main/python/events_analysis/analyze_events.py:   File "src/main/python/events_analysis/analyze_events.py", line 3, in <module>
Process Handler Stdout for Python Script: src/main/python/events_analysis/analyze_events.py:     import plotly.graph_objects as go
14:24:26.375 ERROR beam.analysis.AnalysisProcessor$ - [4] Process Handler Stderr for Python Script: src/main/python/events_analysis/analyze_events.py: ImportError: No module named plotly.graph_objects
Process Handler Stdout for Python Script: src/main/python/events_analysis/analyze_events.py: ImportError: No module named plotly.graph_objects`

Comment (nikolayilyin) [reply build.gradle:614]: here is the log of the docker build:
[beam-image-creation.log](https://github.com/LBNL-UCB-STI/beam/files/6125839/beam-image-creation.log)

Comment (JustinPihony) [reply build.gradle:614]: @nikolayilyin I tried to fix this by pulling in python2 also. However there are a lot of hurdles I had to overcome. Right now we need https://github.com/pysal/mapclassify/pull/110 to be merged and released. I mean we can pull the github repo with the PR, but that opens even more cans of worms. Then, beyond that, many libraries are not guaranteeing or even supporting python 2 much any longer. So, my take is that we run everything in python 3. Is there a reason we are using python 2 at all? If not then let's remove that need and simplify things?

Comment (nikolayilyin) [reply build.gradle:614]: As discussed during daily stand-up, there are no reasons to stay on python 2

**Comment by JustinPihony** [comment]
Superceded by https://github.com/LBNL-UCB-STI/beam/pull/3148 - shall I close?

**Comment by nikolayilyin** [comment]
there is still an issue with orca
[issue-with-orca.txt](https://github.com/LBNL-UCB-STI/beam/files/6206811/issue-with-orca.txt)

**Comment by nikolayilyin** [comment]
also, there is a new documentation entry and a simple but useful shell script to run the beam docker image

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
ooks like flaky tests .. everything passing locally

**Comment by nikolayilyin** [comment]
test!

**Review by JustinPihony — APPROVED** [review]

**Pull request merged**